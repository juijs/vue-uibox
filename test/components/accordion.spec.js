import { mount } from 'vue-test-utils'
import UixAccordionBox from '@/components/accordion'

describe('accordion.js', () => {
    const wrapper = mount(UixAccordionBox, {
        propsData: {
            items: [
                "Group Item #1",
                "Group Item #2",
                "Group Item #3"
            ],
            index: 1
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})