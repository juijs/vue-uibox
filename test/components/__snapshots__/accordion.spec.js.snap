// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accordion.js snapshot!! 1`] = `
<div
  class="accordion"
  style="width: 100%;"
>
  <div
    class="title"
  >
    
        Group Item #1 
    <i
      class="icon-arrow1"
    />
  </div>
  <div
    class="title active"
  >
    
        Group Item #2 
    <i
      class="icon-arrow1"
    />
  </div>
  <div
    class="content"
  />
  <div
    class="title"
  >
    
        Group Item #3 
    <i
      class="icon-arrow1"
    />
  </div>
   
</div>
`;
